Cargo.toml

```diff
@@ -23,3 +23,27 @@ members = [
 ]
 
 exclude = ["gir", "checker"]
+
+[workspace.package]
+homepage = "https://gtk-rs.org/"
+authors = ["The gtk-rs Project Developers"]
+version = "0.19.0"
+repository = "https://github.com/gtk-rs/gtk-rs-core"
+license = "MIT"
+exclude = [
+    "gir-files/*",
+]
+edition = "2021"
+rust-version = "1.70"
+
+[workspace.dependencies]
+once_cell = "1.19"
+libc = "0.2"
+bitflags = "2.4"
+thiserror = "1"
+gir-format-check = "^0.1"
```

Comment:
This should be in build dependencies probably?

Reply:
You can't define build dependencies inside a workspace's Cargo.toml; crates have to use `gir-formart-check.worskspace = true`

Reply:
dev-dependencies

Reply:
such key doesn't exist :/ you just define everything the main Cargo.toml and opt-in in sub-crates

Reply:
Ok then :) Nothing we can do!